packages/first-boot-wizard/files/usr/lib/lua/firstbootwizard.lua

```diff
@@ -318,47 +197,66 @@ function check_lock_file()
     return true
 end
 
+-- Remove lock file
 function remove_lock_file()
-    execute("rm /etc/first_run")
+    utils.execute("rm /etc/first_run")
+end
+
+-- List downloaded lime-defaults
+function read_configs()
+    local tempFiles = fs.dir("/tmp/")
+    local result = {}
+    for file in tempFiles do
+        if (file ~= nil and file:sub(1, 12) == "lime-default") then
+            local ap = getAp(file)
+            table.insert(result, {
+                ap = string.gsub(ap, "\"", ""),
+                file = file
+            })
+        end
+    end
+    return result
 end
 
+-- Create new network
 function apply_user_configs(configs)
+    -- Mesh network name
     local name = configs.ssid
+    -- Save changes in lime-defaults
     local uci_cursor = uci.cursor()
-    -- nixio.syslog("crit", "FBW apply_user_configs ssid "..ssid)
     uci_cursor:set("lime-defaults", 'wifi', 'ap_ssid', name)
     uci_cursor:set("lime-defaults", 'wifi', 'apname_ssid', name..'/%H')
     uci_cursor:set("lime-defaults", 'wifi', 'adhoc_ssid', 'LiMe.'..name..'/%H')
     uci_cursor:set("lime-defaults", 'wifi', 'ieee80211s_mesh_id', 'LiMe.'..name..'/%H')
     uci_cursor:commit("lime-defaults")
-
-    -- Apply config and reboot
-    execute("rm /etc/config/lime")
+    -- Clean previus lime configuration
     clean_lime_config()
-    execute("/rom/etc/uci-defaults/91_lime-config")
-    execute("rm /etc/first_run")
+    -- Apply new configuration and reboot
+    utils.execute("/rom/etc/uci-defaults/91_lime-config")
```

Comment:
it is not clear if this is what applies the config or the line two lines below. could you clarify?

Reply:
Changed on 8e3e4490dbc7d0a73ff30e764700bab72760e91d. I remove line 232 (Clean...) and move clean_lime_config() after the comment section.